### PR TITLE
Always call sns get proposal on details page

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

After the recent Sns Governance behavior change, `list_proposals: payload_text_rendering` is no longer returned as part of a proposal. This breaks the `nns-dapp`, causing the Sns payload to not be rendered under certain conditions.

To reproduce:
- Sign in
- Open any Sns proposals page.
- Wait about 10 seconds to ensure a certified version of the `list_proposals` response is received.
- Click on any proposal card to see its details.
- The proposal payload is not rendered.

# Changes

- Make `getSnsProposalById` to not use the `snsProposalsStore`.
- Removed `getProposalFromStoreById`.

# Tests

- Updated.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/pages/SnsProposalDetail.svelte [hunk 1]
```diff
@@ -145,21 +140,13 @@
         updating = true;
 
         await Promise.all([
-          // skip neurons call when not signedIn or when neurons are ready
+          // skip neurons call when not signedIn or when neurons are ready available
```
Comment (dskloetd) [inline frontend/src/lib/pages/SnsProposalDetail.svelte:143]: Was this change intended? It reads strange.

File: frontend/src/lib/pages/SnsProposalDetail.svelte [hunk 2]
```diff
@@ -145,21 +140,13 @@
...
           !$authSignedInStore ? undefined : loadSnsParameters(universeId),
         ]);
-        /*
-        Reload proposal only after `syncSnsNeurons` is done,
-        to be sure that the `reloadForBallots` flag is calculated correctly (based on user neuron presence).
-        When `reloadForBallots` is true, the proposal will be reloaded regardless of other conditions.
-        Otherwise, the proposal from snsProposalsStore will be used when:
-          - proposal data in the store is certified
-          - and user has no neurons for selected Sns.
-         */
```
Comment (dskloetd) [inline frontend/src/lib/pages/SnsProposalDetail.svelte:162]: The comment is removed, but the line the comment was on is still there.
Which code was the comment referring to and why is no nearby code changed?

File: frontend/src/lib/pages/SnsProposalDetail.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/pages/SnsProposalDetail.svelte]: Thanks, I misread it, reverted.

File: frontend/src/lib/pages/SnsProposalDetail.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/pages/SnsProposalDetail.svelte:162]: The code explained the approach behind the `reloadForBallots` argument - https://github.com/dfinity/nns-dapp/pull/4998/files#diff-174be69a52ffaccb8651075fe48e038b5e2cade7f6fc55e58dcdbccc98bc8e5eL111. The argument has been removed.

**Review by dskloetd — APPROVED** [review]
